ansible/roles/debops.nsswitch/templates/etc/ansible/facts.d/nsswitch.fact.j2

```diff
@@ -32,7 +32,7 @@ try:
                         service_actions.update({ words[i]: words[i+1]})
                 service_list = []
                 for service in words[1:]:
-                    if service in service_actions.keys():
+                    if list(service in service_actions.keys():
```

Comment:
Here be dragons (missing closing parentheses). Also, I don’t think `list` is required here.

Reply:
Good catch! The GitLab CI test environment didn't catch this, perhaps it's a good idea to add an Ansible facts check after the playbook completes.

Reply:
oh, this is not only the missing bracket but the "keys()" part should be removed as well.